Solutions/Microsoft Exchange Security - Exchange Online/Package/mainTemplate.json

```diff
@@ -779,7 +780,7 @@
                 "[variables('parserObject1')._parserId1]"
               ],
               "properties": {
-                "parentId": "[resourceId('Microsoft.OperationalInsights/workspaces/savedSearches', parameters('workspace'), 'Parser-for-ExchangeConfiguration')]",
+                "parentId": "[resourceId('Microsoft.OperationalInsights/workspaces/savedSearches', parameters('workspace'), 'Parser for ExchangeConfiguration')]",
```

Comment:
A Bug on Solution installation was corrected by PG introducing this type of parentID instead of the one with Space. Are you sure that this revert back to name with space will not brake again the solution installation process ?